samples/CS/XAMLBehaviorsSample/DataTriggerControl.xaml.cs

```diff
@@ -17,11 +19,55 @@
 
 namespace XAMLBehaviorsSample
 {
-    public sealed partial class DataTriggerControl : UserControl
+    public enum StateEnum { StateOne, StateTwo, StateThree };
+    public sealed partial class DataTriggerControl : UserControl, INotifyPropertyChanged
     {
+        private StateEnum _state;
+
+        public event PropertyChangedEventHandler PropertyChanged;
+        // This method is called by the Set accessor of each property.
+        // The CallerMemberName attribute that is applied to the optional propertyName
+        // parameter causes the property name of the caller to be substituted as an argument.
+        private void NotifyPropertyChanged([CallerMemberName] String propertyName = "")
+        {
+            if (PropertyChanged != null)
```

Comment:
Not thread safe, assign a variable directly or use null propagation 
```
var handler = PropertyChanged;
if(handler != null)
{
    handler(this, new PropertyChangedEventArgs(propertyName);
}
```
Or
```
PropertyChanged?.Invoke(this, new PropertyChangedEventArgs(propertyName));
```

Reply:
Resolved in ff34408